web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/TrialsApiController.java

```diff
@@ -151,12 +163,31 @@ public ResponseEntity<Map<String, List<Trial>>> trialsGetByCancerTypes(
         return new ResponseEntity<>(result, status);
     }
 
-    private Tumor getTumor(JSONObject jsonObject, String oncoTreeCode) {
+    private Tumor getTumor(JSONObject jsonObjectOncotree, JSONObject jsonObjectTrials, String oncoTreeCode) {
         Tumor tumor = new Tumor();
-        if (jsonObject.containsKey(oncoTreeCode)) {
-            JSONObject tumorObj = (JSONObject) jsonObject.get(oncoTreeCode);
+        if (jsonObjectOncotree.containsKey(oncoTreeCode)) {
+            JSONObject tumorObj = (JSONObject) jsonObjectOncotree.get(oncoTreeCode);
             Gson gson = new Gson();
+            // System.out.println(tumorObj.toString());
             tumor = gson.fromJson(tumorObj.toString(), Tumor.class);
+
+            List<Object> trialIds = (List<Object>) tumorObj.get("trials");
```

Comment:
This is not only ids? The naming should be updated.

Reply:
At this point the trial objects do not have any other information besides IDs but yes the naming does not make sense. I will update this